Tests/Utilities/ToggleFactory.swift

```diff
@@ -64,13 +70,25 @@ class ToggleFactory {
                metadata: Metadata(description: "", group: ""))
     }
     
+    func objetEmptyMetadataToggle(_ value: Object) -> Toggle {
+        Toggle(variable: "object_toggle",
+               value: .object(value),
+               metadata: Metadata(description: "", group: ""))
+    }
+    
     func makeToggles() -> [Toggle] {
         [
             booleanToggle(true),
             integerToggle(42),
             numericToggle(3.1416),
             stringToggle("Hello World"),
-            secureToggle("YXe+Ev76FbdwCeDCVpZNZ1RItWZwKTLXF3/Yi+x62n3JWbvPo6YK")
+            secureToggle("YXe+Ev76FbdwCeDCVpZNZ1RItWZwKTLXF3/Yi+x62n3JWbvPo6YK"),
+            objetToggle(Object(map: [
```

Comment:
maybe worth adding this to the other makeToggles functions? i.e `makeTogglesWithNoMetadata`, `makeToggles(count: Int) `

Reply:
Which other `makeToggles` functions do you mean? I have added two functions `objetEmptyMetadataToggle` and `objetToggle`, anything I missed?

Reply:
Resolving conversation as pr updated ✅